conversational-skills/README.md

```diff
@@ -0,0 +1,304 @@
+--header 'Content-Type: application/json' \
+--header 'Authorization: {{AUTHORIZATION}}' \
+--data '{
+    "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+    "specification": {
+        "servers": [
+            {
+                "url": "https://myProCodeProvider.com"
+            }
+        ],
+        "components": {
+            "securitySchemes": {
+                "authentication_method": "basic",
+                "basic": {
+                    "username": {
+                        "type": "value",
+                        "value": "myBasicUserName"
+                    }
+                }
+            }
+        }
+    },
+    "private": {
+        "authentication": {
+            "basic": {
+                "password": {
+                    "type": "value",
+                    "value": "myBasicPassword"
+                }
+            }
+        }
+    }
+}'
+```
+
+**Example Request Body**
+```json
+{
+    /* A user-provided ID that should not have spaces, tabs, or special characters. */ 
+    "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+    "specification": {
+        "servers": [
+            {
+                /* URL where the client-implemented endpoints are hosted. */ 
+                "url": "https://myProCodeProvider.com"
+            }
+        ],
+        "components": {
+            "securitySchemes": {
+                /* Required enum that defines the type of authentication needed. */
+                "authentication_method": "basic",
+                /* Non-sensitive information pertaining to a specific authentication method */
+                "basic": {
+                    "username": {
+                        "type": "value",
+                        "value": "myBasicUserName"
+                    }
+                }
+            }
+        }
+    },
+    /* Sensitive, private data for authentication. */
+    "private": {
+        "authentication": {
+            "basic": {
+                "password": {
+                    "type": "value",
+                    "value": "myBasicPassword"
+                }
+            }
+        }
+    }
+}
+```
+
+**Example Response**
+
+```json
+{
+  "provider_id": "myProCodeProvider-noSpaces_no_special_chars",
+  "specification": {
+    "servers": [
+      {
+        "url": "https://myProCodeProvider.com"
+      }
+    ],
+    "components": {
+      "securitySchemes": {
+        "basic": {
+          "username": {
+            "type": "value",
+            "value": "myBasicUsername"
+          }
+        },
+        "authentication_method": "basic"
+      }
+    }
+  }
+}
+```
+
+- Currently, we support the registration of only **one** conversational skill provider. 
+- Information within the `private` object will not be returned by the API on response.
+- `authentication_method`: An required enum in the request body denoting the authentication method for the conversational skill provider.
```

Comment:
I think it is worth mentioninig/referencing the testitall starter kit for examples on auth options otherwise how do we expect them to guess something like 
``` 
"api_key": {
                "My-API-Key": {
                    "type": "value",
                    "in": "header",
                    "value": "myAPIKeyAuth"
                }
            }
            ```

Reply:
I want to keep this lo fi for now, and then have links to the cloud docs when that is ready